lib/power-after-fail-lib.robot

```diff
@@ -0,0 +1,28 @@
+*** Settings ***
+Documentation       Collection of keywords related to the Power State After
+...                 Power Fail option
+
+
+*** Keywords ***
+Enter Power Management Options
+    [Documentation]    This keyword enters the Power Management Options
+    ...    from the Dasharo System Features submenu.
+    ${menu_construction}=    Get Setup SubMenu Construction
+    ${system_index}=    Get Index From List    ${menu_construction}    Power Management Options
+    Press Key N Times And Enter    ${system_index}    ${ARROW_DOWN}
+
+Set Power After Fail State
+    [Documentation]    Sets the Power State After Power Fail option to a
+    ...    desired state declared in ${state}. Available options are:
+    ...    Powered Off
+    ...    Powered On
+    ...    The state at the moment of failure
+    [Arguments]    ${state}
+    ${menu_construction}=    Get Setup SubMenu Construction
```

Comment:
No need for dedicated keyword just for that. The `Set Option Value` can be directly used in tests like here: https://github.com/Dasharo/open-source-firmware-validation/blob/rework-terminal-kwds/dasharo-security/me-neuter.robot#L50